components/headers/H1.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getString } from "../../lib/richText";
+
+const H1Style = styled.h1`
+    font-size: 80px;
+    font-family: cormorant-garamond, serif; 
```

Comment:
Can you please start a `typography.ts` file in a directory either called `variables/` (other types of variable files like `colors.ts` and stuff can go here later) or `style/` that contains type sizes, font families, etc? This way everything is centralized.  Then in the component you'd have `font-size: ${variables.h1}`.

Also, take a look at [clamp](https://css-tricks.com/linearly-scale-font-size-with-css-clamp-based-on-the-viewport/). Eventually you'll have to scale the type somehow and this can be a good and fluid way to do it.